learn/configuration/instance_options.md

```diff
@@ -169,15 +171,52 @@ Sets the directory where Meilisearch will create dump files.
 
 ### Import dump
 
+::: warning
+This is a CLI flag and does not take any values. Assigning a value will throw an error.
+:::
+
 **Environment variable**: N/A
 **CLI option**: `--import-dump`
-**Default value**: `none`
-**Expected value**: a filepath pointing to a `.dump` file
+**Default**: Disabled
 
-Imports the dump file located at the specified path. Path must point to a `.dump` file.
+Replaces `data.ms` with dump data and starts the server using the provided dump. Path must point to a `.dump` file.
 
 Meilisearch will only launch once the dump data has been fully indexed. The time this takes depends on the size of the dump file.
 
+If the flag is specified when a database exists, an error occurs in the CLI.
+
+*This option is not available as an environment variable.*
+
+### Ignore missing dump
+
+::: warning
+This is a CLI flag and does not take any values. Assigning a value will throw an error.
+:::
+
+**Environment variable**: N/A
+**CLI option**: `--ignore-missing-dump`
+**Default**: Disabled
+
+Prevents a Meilisearch instance from throwing an error when `--import-dump` does not point to a valid dump file. Meilisearch will then continue its process and not import any dump.
+
+If the `--ignore-missing-dump` flag is not specified, and the file cannot be found, an error occurs in the CLI.
+
+*This option is not available as an environment variable.*
+
+### Ignore dump if DB exists
+
+::: warning
+This is a CLI flag and does not take any values. Assigning a value will throw an error.
+:::
+
+**Environment variable**: N/A
+**CLI option**: `--ignore-dump-if-db-exists`
+**Default**: Disabled
+
+Prevents a Meilisearch instance with an existing database from throwing an error when using `--import-dump`. Instead, the dump will be ignored and Meilisearch will launch using the existing database.
+
+This command will throw an error if `--import-dump` is not defined.
+
 *This option is not available as an environment variable.*
```

Comment:
This one might have been addressed on a PR on the main branch, so perhaps it's not necessary?

```suggestion
_This option is not available as an environment variable._
```